packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -444,7 +444,7 @@ Generated by [AVA](https://ava.li).
       │ 2. onBuild    ↓ │ build.command from netlify.toml␊
       └─────────────────┘ ␊
       ┌─────────────────┐␊
-      │ 3. onBuild      │ Plugin @netlify/plugin-functions-core␊
+      │ 3. onBuild    ↓ │ Plugin @netlify/plugin-functions-core␊
```

Comment:
This change is due to a small problem fixed by https://github.com/netlify/build/pull/2054.

Reply:
Solved by rebase